src/js/movies.js

```diff
@@ -1,3 +1,87 @@
+import  movies  from   '../data/movies.json' assert { type: 'json' };
+
+console.log('movies2', movies) 
+
+let moviesList = document.querySelector('.movies-now')
+let futureMovies = document.querySelector('.movies-later ')
+
+
+class Movies {
+    constructor(title,description,country,languages,director, actors, posterUrl,imageUrl,videoUrl,genres,ageRestriction,releaseDate,screeningTimes,tags,rating){
+        this.title = title;
+        this.description = description;
+        this.country=country;
+        this.languages = languages;
+        this.director = director;
+        this.actors = actors;
+        this.posterUrl = posterUrl;
+        this.imageUrl = imageUrl;
+        this.videoUrl = videoUrl;
+        this.genres = genres;
+        this.ageRestriction = ageRestriction;
+        this.releaseDate = releaseDate;
+        this.screeningTimes = screeningTimes;
+        this.tags = tags;
+        this.rating = rating;     
+
+    }
+
+
+     static displayMovies(){
+        movies.movies.forEach(movie =>{
+      if(new Date(movie.releaseDate ).getFullYear() === new Date().getFullYear()){
+                  // get the heading
+       let heading = document.querySelector('.heading-1')
+         heading.innerText = 'På Bion Just Nu' 
+         Movies.createMovie(movie)
+         
+                
+            }else{
+            let heading = document.querySelector('.heading-2')
+            heading.innerText = 'kommer snårt'
```

Comment:
A suggestion of change to "Kommer Snart" so it's more consistent to "På Bio Just Nu" :)

Reply:
Can you be more explicit? I am not sure I understand what you mean

Reply:
I'm referring to the typo in the string from heading.innerText = 'kommer snårt'. :) 
And also the string above it ("På Bio Just Nu") has a capital letter on every word. Only a small detail.